test/onnx/test_utility_funs.py

```diff
@@ -780,6 +780,32 @@ def forward(self, x):
         ort_outs2 = ort_sess.run(None, ort_inputs)
         [np.testing.assert_allclose(ort_out1, ort_out2, atol=1e-7, rtol=0.001) for ort_out1, ort_out2 in zip(ort_outs1, ort_outs2)]
 
+    def test_onnx_function_substitution_pass(self):
+        class MyModule(torch.nn.Module):
+            def __init__(self):
+                super(MyModule, self).__init__()
+
+            @torch.jit.script
```

Comment:
Thanks for this fix!
There's a flake8 error about this method:
"Invalid first argument 'x' used for instance method. Use the canonical first argument name in methods, i.e. self"

If this is blocking, I guess we can use the non-member class example for test.

Reply:
Thanks. I fixed both errors.
Interestingly, this error does not seem to be coming from flake8, but some other tool. Anyway it should be fixed now.